opnode/flags/flags.go

```diff
@@ -31,6 +31,30 @@ var (
 		Required: true,
 		EnvVar:   prefixEnvVar("ROLLUP_CONFIG"),
 	}
+	L2NodeAddr = cli.StringFlag{
+		Name:     "eth.l2",
+		Usage:    "Address of L2 User JSON-RPC endpoint to use (eth namespace required)",
+		Required: true,
+		EnvVar:   prefixEnvVar("L2_ETH_RPC"),
+	}
+	RPCListenAddr = cli.StringFlag{
+		Name:     "rpc.addr",
+		Usage:    "RPC listening address",
+		Required: true,
+		EnvVar:   prefixEnvVar("RPC_ADDR"),
+	}
+	RPCListenPort = cli.IntFlag{
+		Name:     "rpc.port",
+		Usage:    "RPC listening port",
+		Required: true,
+		EnvVar:   prefixEnvVar("RPC_PORT"),
+	}
+	WithdrawalContractAddr = cli.StringFlag{
+		Name:     "rpc.withdrawalcontractaddress",
+		Usage:    "Address of the Withdrawal contract. Used to serve RPC",
+		Required: true,
+		EnvVar:   prefixEnvVar("WITHDRAWAL_CONTRACT_ADDR"),
+	}
```

Comment:
This is the L2 contract address we're proving the state of right? 
If so, then it is currently spec'd to be a predeploy at `0x4200000000000000000000000000000000000015`. 

I'd propose either: 
1. hardcode it in which would likely simplify some code.
2. make the flag optional and default to the the above address.

Reply:
Yup, that's the L2 contract address. I'll make it optional with the default so the contract itself can be mocked for integration testing if needed.